RULES.md

```diff
@@ -68,7 +68,7 @@ Note that the notice ID naming conventions changed in `v2` to make contributions
 |                           | [W006](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#W006)        	| Missing route short name                                                        	|
 |                           | [W007](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#W007)        	| Missing route long name                                                         	|
 |                           | [W008](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#W008)        	| Route long name contains short name                                             	|
-|                           | [W009](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#W009)        	| Dataset should cover at least the next 30 days of service                       	|
+|[`FeedExpirationDateNotice`](#FeedExpirationDateNotice)| [W009](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#W009)        	| Dataset should cover at least the next 30 days of service                       	|
```

Comment:
For multiple rules in v1 that are being condensed under v2, I'd only list one row in the table that references both rules. IMHO our goal should be to have a compact table for v2.

So something like this:

| Error ID (v2.0+) | Error ID (v1.x) 	| Error Title                                                             	|
|--------------------------	|--------------------	|-------------------------------------------------------------------------	|
|[`FeedExpirationDateNotice`](#FeedExpirationDateNotice)| [W009](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#W009), [E040](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E040)        	| Dataset should cover at least the next 30 days of service. Dataset should be valid for at least the next 7 days.